recipes/spirv-cross/all/conanfile.py

```diff
@@ -18,6 +18,7 @@ class SpirvCrossConan(ConanFile):
     generators = "cmake"
     settings = "os", "arch", "compiler", "build_type"
     options = {
+        "exceptions": [True, False],
```

Comment:
keep shared and fPIC options in first positions please, for consistency among CCI recipes.